tests/classes/aioseop-notices/test-delay-time.php

```diff
@@ -90,9 +90,11 @@ protected function mock_notice() {
 	 * Override and skip.
 	 *
 	 * @since 3.0
+	 *
+	 * @dataProvider data_screens
 	 */
-	public function test_enqueue_scripts_on_screens() {
-		$this->markTestSkipped('Skip');
+	public function test_enqueue_scripts_on_screens( $screen_id, $url, $dir ) {
+		$this->markTestSkipped( 'Skip' );
```

Comment:
maybe markTestIncomplete would be a better choice? same for the ones below. Skipped is better used for a particular condition, say dependency on PHP version.